app/decorators/offence_decorator.rb

```diff
@@ -23,4 +30,12 @@ def plea_sentence(plea)
       plea: plea.code&.humanize || t('generic.not_available'),
       pleaded_at: plea.pleaded_at&.to_date&.strftime('%d/%m/%Y') || t('generic.not_available'))
   end
+
+  def mode_of_trial_reason_descriptions
+    mode_of_trial_reasons.map { |mode_of_trial| mode_of_trial_reason_description(mode_of_trial) }
```

Comment:
```suggestion
    mode_of_trial_reasons.map { |mode_of_trial_reason| mode_of_trial_reason_description(mode_of_trial_reason) }
```
or, if preferred, for brevity
```suggestion
    mode_of_trial_reasons.map { |reason| mode_of_trial_reason_description(reason) }
```

Reply:
perhaps for simplicity we do not need the other method 
```suggestion
    mode_of_trial_reasons.map { |reason| reason.description ||  t('generic.not_available') }
```

Reply:
👍 yes definitely. For brevity I have followed the suggestion to remove the mode_of_trial_reason_descriptions method which resolves the naming issue.